exporting/aws_kinesis/metadata.yaml

```diff
@@ -0,0 +1,173 @@
+# yamllint disable rule:line-length
+---
+id: 'export-aws-kinesis'
+meta:
+  name: 'AWS Kinesis'
+  link: 'https://graphite.readthedocs.io/en/latest/'
```

Comment:
Hello, 

I am reviewing again to be sure I did not miss and I found that we are using a `graphite` link here.